apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -85,6 +85,25 @@ type TempoStackSpec struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Tenants Configuration"
 	Tenants *TenantsSpec `json:"tenants,omitempty"`
+
+	Observability ObservabilitySpec `json:"observability,omitempty"`
```

Comment:
wdyt about this @iblancasa @pavolloffay ?

Reply:
I think it's OK. It's generic enough to add new fields later if needed.